client/my-sites/plan-features-2023-grid/plans-2023-tooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled';
+import { TranslateResult } from 'i18n-calypso';
+import { PropsWithChildren, useRef, useState } from 'react';
+import Tooltip from 'calypso/components/tooltip';
+
+const HoverAreaContainer = styled.span``;
+
+const StyledTooltip = styled( Tooltip )`
+	&.tooltip.popover .popover__inner {
```

Comment:
I guess the Tooltip component wasn't built flexibly enough to be restyled heavily. Ideally, we shouldn't be referencing external classes and structure. It creates coupling / these may change, esp. if component moves to a package, etc. It doesn't look like something we can address. I recall battling to avoid this (the need to reference on the consumer end external classes) when working on another package (TourKit). 🤷 

On another end, it's also not bad, and perhaps we shouldn't really be modifying some things too heavily (like background colors, borders, font-weight, etc.). Having consistent styling for things like tooltips, popovers (etc.) across the app may be a good thing? cc @lucasmendes-design